www/offline.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!--<meta http-equiv="Content-Security-Policy" content="default-src 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *">-->
+    <meta name="format-detection" content="telephone=no">
+    <meta name="msapplication-tap-highlight" content="no">
+    <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width">
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="css/offline.css">
+    <title>Stored Survey</title>
+    <script src="cordova.js"></script>
+
+    <script src="./js/jquery-1.11.3.min.js"></script>
+    <script src="./js/offline.js"></script>
+
+</head>
+<body>
+<div class='header'>
+    <a href = "index.html">
+        <div class='left-align'>
+            <span class="back-button glyphicon glyphicon-menu-left"></span>
+            <img src='pictures/ic_launcher.png' height='25' width='25'>
+            Back to Servey
```

Comment:
Should we expect to go "back" to survey page? I think the back button should log out from the current account. Right now this is fine, but we will have to change it later